src/web/src/pages/signup.tsx

```diff
@@ -0,0 +1,271 @@
+    },
+    imageContainer: {
+      minHeight: '100vh',
+      width: '100vw',
+      position: 'absolute',
+      top: '0',
+      bottom: '0',
+      zIndex: -1,
+      [theme.breakpoints.down(600)]: {
+        display: 'none',
+      },
+    },
+    signUpContainer: {
+      margin: '1% 0 1% 0',
+      display: 'grid',
+      gridTemplateRows: '10% auto 15%',
+      gridGap: '2%',
+      justifyItems: 'center',
+      fontFamily: 'Spartan',
+      height: '510px',
+      width: '510px',
+      padding: '1%',
+      borderRadius: '5px',
+      boxShadow: '2px 4px 4px 1px rgba(0, 0, 0, 0.1)',
+      background: '#ECF5FE',
+      '@media (max-height: 500px) and (max-width: 1024px)': {
+        margin: '0 0 65px 0',
+      },
+      [theme.breakpoints.down(600)]: {
+        background: 'none',
+        boxShadow: 'none',
+        minHeight: '650px',
+        height: '600px',
+        position: 'absolute',
+        top: '0px',
+        width: '100%',
+        margin: '0',
+        padding: '0',
+        gridTemplateRows: '8% auto 17%',
+      },
+    },
+    title: {
+      color: '#121D59',
+      fontSize: '22px',
+    },
+    infoContainer: {
+      width: '100%',
+      position: 'relative',
+    },
+    buttonsWrapper: {
+      margin: '0 auto',
+      width: '50%',
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+    },
+    button: {
+      height: '4rem',
+      width: '40%',
+      fontSize: '1.1em',
+      padding: '0.7em',
+      margin: '5px',
+      background: '#E0C05A',
+      '&:hover': {
+        color: 'black',
+        background: '#EBD898',
+      },
+    },
+    buttonLogin: {
+      height: '4rem',
+      width: '40%',
+      fontSize: '1.1em',
+      padding: '0.7em',
+      margin: '5px',
+      background: '#FF0000',
+      color: '#FFF',
+      '&:hover': {
+        background: '#FF7070',
+      },
+    },
+    text: {
+      textAlign: 'center',
+      fontSize: '0.9em',
+      color: '#474747',
+    },
+  })
+);
+
+const SignUpPage = () => {
+  const classes = useStyles();
+  const [activeStep, setActiveStep] = useState<number>(0);
+  const currentSchema = formSchema[activeStep];
+  const { user, token, login } = useAuth();
+  const [loggedIn, setLoggedIn] = useState(false);
+
+  const handleNext = () => {
+    setActiveStep(activeStep + 1);
+  };
+
+  const handlePrevious = () => {
+    setActiveStep(activeStep - 1);
+  };
+
+  useEffect(() => {
+    if (user) {
+      setLoggedIn(true);
+    }
+  }, [user]);
+
+  const handleSubmit = async (values: SignUpForm, actions: FormikHelpers<SignUpForm>) => {
+    if (activeStep === 4) {
+      try {
+        const { firstName, lastName, email, displayName, github, feeds } = values;
+        const telescopeUser = {
+          firstName,
+          lastName,
+          email,
+          displayName,
+          github,
+          feeds,
+        };
+        const response = await fetch(`${usersServiceUrl}/${user?.id}`, {
```

Comment:
@humphd @DukeManh 
After #2154 we need to change this fetch to the new route right?

Reply:
Yeah, it would be /register so the token is updated.

Reply:
OKay. Let's keep track of #2154.